package.json

```
@@ -39,6 +43,75 @@
     "test": "mocha ./test/*.mjs",
     "test:performance": "mocha test/performance/*.mjs"
   },
+  "release": {
+    "branches": [
+      "main",
+      {
+        "name": "beta",
+        "prerelease": true
+      }
+    ],
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      [
+        "@google/semantic-release-replace-plugin",
+        {
+          "replacements": [
+            {
+              "files": [
+                "src/constants.mjs"
+              ],
+              "from": "export const VERSION = '.*'",
+              "to": "export const VERSION = '${nextRelease.version}'",
+              "results": [
+                {
+                  "file": "src/constants.mjs",
+                  "hasChanged": true,
+                  "numMatches": 1,
+                  "numReplacements": 1
+                }
+              ],
+              "countMatches": true
+            }
+          ]
+        }
+      ],
+      "@semantic-release/changelog",
+      "@semantic-release/npm",
+      [
+        "@semantic-release/git",
+        {
+          "assets": [
+            "src",
+            "dist",
+            "package.json",
+            "CHANGELOG.md"
+          ],
+          "message": "chore(release): ${nextRelease.version}\n\n${nextRelease.notes}"
+        }
+      ],
+      [
+        "@semantic-release/github",
+        {
+          "assets": [
+            {
+              "path": "dist/imgix-core-js.js",
+              "label": "Standalone UMD browser build"
+            },
+            {
+              "path": "dist/imgix-core-js.cjs",
+              "label": "CommonJS build"
+            },
+            {
+              "path": "dist/imgix-core-js.mjs",
+              "label": "ES Module build"
+            }
+          ]
+        }
+      ]
```

To prevent the GH issue spamming that happened previously, two things need to change:

1. Update the latest git tag to include a `v` at the start so semantic release calculates the right version.
2. Update the `@semantic-releate/github` config to not comment on issues and PRs for the first release (this should be removed after the first release). More info here: https://github.com/semantic-release/github

```suggestion
      [
        "@semantic-release/github",
        {
          "successComment": false,
          "releasedLabels": false,
          "assets": [
            {
              "path": "dist/imgix-core-js.js",
              "label": "Standalone UMD browser build"
            },
            {
              "path": "dist/imgix-core-js.cjs",
              "label": "CommonJS build"
            },
            {
              "path": "dist/imgix-core-js.mjs",
              "label": "ES Module build"
            }
          ]
        }
      ]
```